docs/plot_compound.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
```

Comment:
Please add a one-line description at the top of every new file.
People will copy & paste it and then it's good to have a one-line description if they find it in their Download folder a month later.
Something like:
```
"""
Example script illustrating compound regions.
"""
```